package.json

```diff
@@ -5,14 +5,18 @@
   "main": "lib/angular/analytics-ng.js",
   "scripts": {
     "build": "wfm-template-build -m 'wfm.analytics.directives'",
-    "watch": "wfm-template-build -w -m 'wfm.analytics.directives'"
+    "watch": "wfm-template-build -w -m 'wfm.analytics.directives'",
+    "test": "grunt eslint"
   },
   "author": "scm.blanc@gmail.com",
   "license": "MIT",
   "dependencies": {
-    "c3": "0.4.11"
+    "c3": "0.4.11",
+    "lodash": "^4.16.4"
```

Comment:
I though it was the correct thing to do as this [line](https://github.com/danbev/raincatcher-analytics/blob/ab86be6d397579ddda34246449ce9e7c8683520a/lib/angular/directive.js#L33) seem to use it (or possibly underscore).